eslint.config.mjs

```diff
@@ -0,0 +1,103 @@
+import eslintConfigPrettier from 'eslint-config-prettier';
+import eslintPluginImport from 'eslint-plugin-import';
+import eslintPluginReact from 'eslint-plugin-react';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+import { jsRules, reactRules } from './eslint.rules.mjs';
+
+export default tseslint.config(
+  {
+    name: 'nlds/ignores',
```

Comment:
We doen steeds erg ons best om niet `nlds` te gebruiken. Kunnen we ook hier `nlds` vermijden?

Reply:
Ja, ik haal het nog helemaal weg omdat het alleen voor te debuggen is.